man/sd_journal_get_cursor.xml

```diff
@@ -91,6 +89,40 @@
     the current entry matches the specified cursor, 0 if it does not
     match the specified cursor or a negative errno-style error code on
     failure.</para>
+
+    <refsect2>
+      <title>Errors</title>
+
+      <para>Returned errors may indicate the following problems:</para>
+
+      <variablelist>
+        <varlistentry>
+          <term><constant>-EADDRNOTAVAIL</constant></term>
+
+          <listitem><para>The journal context is currently not positioned on any entry, and hence no cursor
+          string can be generated.</para>
+
+          <xi:include href="version-info.xml" xpointer="v209"/></listitem>
```

Comment:
I do not think it is necessary to mention version history for error code.

Reply:
well, i would not have added it, but the CI wouldn't let it pass without. So I added it...

Reply:
Then, why not to add the entry in the deny list?